Update attachment_docusign_suspicious_links.yml

peterdj45 opened the pull request:
# Description

adding logic to flag the word DocuSign with the `i` omitted (DocuSgn) 

also added logic to negate FPs caused by OCR missing characters

# Associated samples
- https://platform.sublime.security/messages/4f8498304aa122e8dd7b31937f69cebc2b07df393f3e58a85f117f0d25451ee9
- https://platform.sublime.security/messages/4f843346280947b2b802ceef8c6c8344b64a234dc2690788ad06eaa464d9607c
- https://platform.sublime.security/messages/4f840d2fd15ec18414350fa9772e4d3b45f5879e5cc37d17ade79387340db5a7
## Associated Hunts
- https://platform.sublime.security/hunts/019acd59-1362-70aa-a1c9-190be003b4c0

peterdj45 committed:
Update attachment_docusign_suspicious_links.yml

IndiaAce commented:
Got some FPs for this 
- https://platform.sublime.security/messages/4f8b823c1cc2c846796f41d23ef096475a775d5c716b5779ed850261267f6fea?preview_id=0199a55d-051b-71f1-b2c1-032087aded0a
- https://platform.sublime.security/messages/4f8b5849adbf51f1b7033096cfb29f4c685e1459b5eb6f998aa80c8959a1eb1b?preview_id=0199a55d-256f-7d2a-9841-bc0f28813d87

alex-herold committed:
Sync .github directory from main branch

- Applied .github directory from main to peter.fn.attachment_sus_docusign_pdf
- Ensures workflows and GitHub configurations are up to date
- Automated sync via script

peterdj45 committed:
Merge branch 'main' into peter.fn.attachment_sus_docusign_pdf

peterdj45 committed:
Update attachment_docusign_suspicious_links.yml

peterdj45 commented:
most of the FPs appear to have been caused by OCR missing a character or two. added logic to account for that

peterdj45 committed:
Update attachment_docusign_suspicious_links.yml

aidenmitchell committed:
Merge branch 'main' into peter.fn.attachment_sus_docusign_pdf
